src/opnsense/service/templates/OPNsense/IDS/suricata.yaml

```diff
@@ -282,43 +285,48 @@ outputs:
             # for the body you need to set app-layer.protocols.smtp.mime.body-md5
             # to yes
             #md5: [body, subject]
+{%      endif %}
 
-        #- dnp3
-        #- ftp
-        #- rdp
-        #- nfs
-        #- smb
-        #- tftp
-        #- ike
-        #- dcerpc
-        #- krb5
-        #- bittorrent-dht
-        #- snmp
-        #- rfb
-        #- sip
-        #- quic
-        #- dhcp:
-            #enabled: yes
+        {{ '#' if 'dnp3' not in OPNsense.IDS.general.eveLog.types.enabled else '' }}- dnp3
```

Comment:
if we only assign valid options in `OPNsense.IDS.general.eveLog.types.enabled` we can iterate over them here:

```
{% for opt in OPNsense.IDS.general.eveLog.types.enabled.split(',') %}
    - {{opt}}
{% endfor %}
```

Use an if statement inside the blob to decorate the option when set.